modules/workload-identity/main.tf

```diff
@@ -64,3 +64,12 @@ resource "google_service_account_iam_member" "main" {
   role               = "roles/iam.workloadIdentityUser"
   member             = local.k8s_sa_gcp_derived_name
 }
+
+
+resource "google_project_iam_member" "workload_identity_sa_bindings" {
+  count = length(var.roles)
```

Comment:
Can we use `for_each`?

Reply:
for each works with maps if I'm not mistaken. I'll need to verify that

Reply:
Done